githubmarkdownui/job.py

```diff
@@ -0,0 +1,27 @@
+from typing import List, Optional
+
+from githubmarkdownui.emoji import Emoji
+
+
+class Job:
+    """This class is intended to be used to help with the creation of job lists or job trees, which can be useful
+    for displaying jobs and child jobs within a build pipeline.
+    """
+    def __init__(self, emoji: Emoji, description: str, info: Optional[str] = None):
```

Comment:
I might be misunderstanding something here, but I assume when creating a `Job` object, callers should be able to pass in a list of tasks, including their status, some links, and other metadata? Is that covered in `info` attribute?

Reply:
Restructured this after our discussion to have separate `CIJob` and `CITask` classes, with separate classes for metadata. Each `CIJob` or `CITask` should be created separately.